easybuild/easyconfigs/f/FFmpeg/FFmpeg-3.3-intel-2017a.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'FFmpeg'
+version = '3.3'
```

Comment:
Because it wasn't there when I made this stuff some weeks ago... 
I'm not going to keep track on all of this.